android/expoview/src/main/java/host/exp/exponent/notifications/ScheduledNotificationReceiver.java

```diff
@@ -27,6 +28,9 @@ public void onReceive(Context context, Intent intent) {
     Bundle bundle = intent.getExtras();
     HashMap details = (HashMap) bundle.getSerializable(KernelConstants.NOTIFICATION_OBJECT_KEY);
     int notificationId = bundle.getInt(KernelConstants.NOTIFICATION_ID_KEY, 0);
+    String schedulerId = (String) details.get(SchedulersManagerProxy.SCHEDULER_ID);
```

Comment:
Shouldn't `SchedulersManagerProxy.SCHEDULER_ID` actually end up in `KernelConstants`?

If not, maybe this line should look like
```java
String schedulerId = SchedulerManager.getSchedulerId(bundle);
```
?